src/prefect/cli/execute.py

```diff
@@ -0,0 +1,63 @@
+import click
+
+import prefect
+from prefect.client import Client
+from prefect.utilities.graphql import with_args
+
+
+@click.group(hidden=True)
+def execute():
+    """
+    Execute flow environments.
+
+    \b
+    Usage:
+        $ prefect execute [OBJECT]
+
+    \b
+    Arguments:
+        cloud-flow  Execute a cloud flow's environment (during deployment)
+
+    \b
+    Examples:
+        $ prefect execute cloud-flow
+
+    \b
+        $ prefect execute local-flow ~/.prefect/flows/my_flow.prefect
+    """
+    pass
+
+
+@execute.command(hidden=True)
+def cloud_flow():
+    """
+    Execute a flow's environment in the context of Prefect Cloud.
+
+    Note: this is a command that runs during Cloud execution of flows and is not meant
+    for local use.
+    """
+    flow_run_id = prefect.context.get("flow_run_id")
+    if not flow_run_id:
+        click.echo("Not currently executing a flow within a cloud context.")
```

Comment:
Could you elevate this to raise an error?  If, for any reason the `flow_run_id` is not set correctly, I'd like the resource manager to be able to log an error instead of a successful exit of the container.